modules/wsg32/0.0.1/MODULE.bazel

```diff
@@ -0,0 +1,5 @@
+module(
+    name = "wsg32",
+    version = "0.0.1",
```

Comment:
Just noticed this project has no official release version, maybe it's better to name this version something like `0.0.1-2025-01-17`? Just in case, in future, there will be an official `0.0.1` release?

Reply:
See https://github.com/bazelbuild/bazel-central-registry/tree/main/modules/upb